src/agent/platform.rs

```diff
@@ -24,7 +24,7 @@ use pb::inventory_service_client::InventoryServiceClient;
 use crate::registry::PkgRef;
 use crate::version::VERSION;
 
-const EXPIRATION_SLACK: Duration = Duration::from_secs(60);
+const EXPIRATION_SLACK: Duration = Duration::from_secs(10*60);
```

Comment:
Why was the expiration changed?

Reply:
Me being paranoid, now that we're seeing session tokens being expired.

Actually it used to be 10 mins but then I made the `server-stub` issue 3 min tokens. So to test the logic better, I changed it to 1 min. But I'd rather be a bit safer now.